README.md

```diff
@@ -202,11 +202,11 @@ Follow the steps to install dependencies on a specific platform:
     $ choco install -y cmake
     ```
 
-4. Install [vcpkg](https://github.com/NordicPlayground/vcpkg.git).
+4. Install [vcpkg](https://github.com/Microsoft/vcpkg.git).
```

Comment:
Question regarding vcpkg, if it points to Microsoft /vcpkg, then we need to make a PR to Microsoft /vcpkg to release pc-ble-driver right? Do we have the control to release it in time?

Reply:
The reason for us having to fork off from MSFT vcpkg repo was that there was an issue with fmtlib.
Since this issue is now fixed and the MSFT vcpkg repo has a tagged version for this we should use this one.

Regarding releasing pc-ble-driver as a vcpkg on MSFT vcpkg we will not be able to control when the PR is approved. But that is something we need to solve in pc-ble-driver-js and similar. We can also use port overlays for this, removing the need to fork the MSFT vcpkg and add the pc-ble-driver.